cc/toolchains/llvm/cc_toolchain_config.bzl

```diff
@@ -77,10 +77,12 @@ def cc_toolchain_config(
         "-fdata-sections",
     ]
 
-    cxx_flags = [
-        # The whole codebase should build with c++14
-        "-std=c++14",
-    ]
+    cxx_flags = select({
```

Comment:
How does this compare to https://github.com/swift-nav/rules_swiftnav/blob/main/cc/defs.bzl#L62 ? I actually wasn't aware of the flag directly in the toolchain file.

Reply:
what what is highlighted here, if you use the bazel `cc_library` or `cc_binary` directly it will impose the "global cxx standard". in the code snippet that [you posted](https://github.com/swift-nav/rules_swiftnav/blob/main/cc/defs.bzl#L62), that only applies to `swift_cc_library` and `swift_cc_binary` targets.